src/app/OperationalDeviceProxy.h

```diff
@@ -132,6 +132,8 @@ class DLL_EXPORT OperationalDeviceProxy : public DeviceProxy, SessionReleaseDele
      */
     CHIP_ERROR Disconnect() override;
 
+    void HandleCommissionerConnected(SessionHandle handle);
```

Comment:
Can you improve the name of this. For example, SetConnectedSession

Reply:
Done.